Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2736,6 +2736,13 @@ static FloatRect computeMarqueeSelectionRect(const WebCore::FloatPoint& point1,
     });
 }
 
+Vector<FloatRect> UnifiedPDFPlugin::selectionBoundsAcrossDocumentInContentSpace(const PDFSelection *selection) const
```

Comment:
Maybe we should have the original method take a destination coordinate space argument so we can just keep everything contained

Reply:
That's a good point. I'll do some refactoring here.

Reply:
Addressed here: https://github.com/WebKit/WebKit/pull/26076